core/src/main/java/com/orange/ouds/core/component/OudsColoredBox.kt

```diff
@@ -83,7 +83,7 @@ fun OudsColoredBox(
 }
 
 /**
- * Contains classes to build an [com.orange.ouds.core.component.coloredbox.OudsColoredBox].
+ * Contains classes to build an [com.orange.ouds.core.component.OudsColoredBox].
  */
 object OudsColoredBox {
```

Comment:
The `Companion` object of `OudsColoredBox.Color` appears in the generated documentation, we should remove it.